backend-demo/README.md

```diff
@@ -0,0 +1,16 @@
+# Portal backend demo
+
+ASHPD enables writing [portal backends](https://docs.flatpak.org/en/latest/portal-api-reference.html#idm45771421251808) that implement a subset of backend interfaces.
+
+This demo provides a backend that registers the `org.freedesktop.impl.portal.desktop.ashpd-backend-demo` name and implements the [Account](https://docs.flatpak.org/en/latest/portal-api-reference.html#gdbus-org.freedesktop.impl.portal.Account), [Screenshot](https://docs.flatpak.org/en/latest/portal-api-reference.html#gdbus-org.freedesktop.impl.portal.Screenshot), [Secret](https://docs.flatpak.org/en/latest/portal-api-reference.html#gdbus-org.freedesktop.impl.portal.Secret), [Settings](https://docs.flatpak.org/en/latest/portal-api-reference.html#gdbus-org.freedesktop.impl.portal.Settings) and [Wallpaper](https://docs.flatpak.org/en/latest/portal-api-reference.html#gdbus-org.freedesktop.impl.portal.Wallpaper) interfaces.
```

Comment:
I wouldn't mention all the portals here, it is too much

Reply:
Removed.